### DISCUSSION TITLE
Scrolling with a mouse moves 3 lines instead of 1

### DISCUSSION THREAD

**Discussion by mtdowling** [zellij-org/zellij#1979] [Q&A]
I'm using Alacritty with Zellij in MacOS, and scrolling with my trackpad in Alacritty can move a single line at a time. This makes scrolling pretty smooth, and has a nice easing effect as it slows down. In Zellij though, it always scrolls at minimum 3 lines at a time, and the easing effect is much choppier. Is there a way to tweak this somehow, or is this typical of multiplexers?

**Comment by dsully** [comment]
I'm seeing this with Kitty as well.

**Comment by evakhoni** [comment]
same issue with Ghostty, Fedora KDE. very very annoying. the kde system scrolling speed have an effect on this (jumps 6 or 9 if i increase it) but below certain point it keeps scrolling 3 lines even if i decrease the system speed to absolute minimum. seems like a bug rather that a feature to me. I wonder can we convert that to issue somehow or I need to open a new one? 
edit: opened https://github.com/zellij-org/zellij/issues/5315
